.github/ISSUE_TEMPLATE/4-xp-bug-report.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug", "triage"]
```

Comment:
This should either be named differently or be merged into the existing bug report.